clustering/train_clusterer.py

```diff
@@ -0,0 +1,162 @@
+import argparse
+import itertools
+
+import numpy as np
+
+import pickle
+from pathlib import Path
+
+import torch
+
+from tqdm.auto import tqdm
+
+from torch.utils.data import DataLoader, IterableDataset
+
+from kmeans_pytorch import KMeans as BalancedKMeans
```

Comment:
had issues with importing KMeans, turned up an error 
but no issues importing kmeans -- did you experience this?

Reply:
Did you use the kmeans from pip or the one from https://github.com/kernelmachine/cbtm?